resolver-dns/src/main/java/io/netty/resolver/dns/DnsNameResolver.java

```diff
@@ -507,6 +512,12 @@ protected void doResolve(String inetHost,
             return;
         }
 
+        if (inetHost.isEmpty()) {
```

Comment:
@normanmaurer - sgtm. I'm not sure if @trustin would have any opinion on this though.